run/helm/snap/README.md

```diff
@@ -0,0 +1,246 @@
+
+This command deploys Snap on the Kubernetes cluster in the default configuration. To learn how to override the default configuration see section [Configuration](#6-configuration).
+
+## 5. Uninstalling the Chart
+
+To uninstall/delete the my-snap daemonset use command:
+
+`$ helm delete my-snap`
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## 6. Configuration
+
+To see the default configuration of Snap chart run:
+
+`$ helm inspect values stable/snap`
+
+To override any of these settings create your own YAML formatted file and pass it during installation:
+```
+$ echo 'host_plugins: /opt/snap' > new_values.yaml
+$ helm  install -f new_values.yaml stable/snap
+```
+or use `set` flag to specify custom values on the command line like this:
+
+`$ helm install --set host_plugins=/opt/snap stable/snap`
+
+You will find more information in Helm Guide section [Customizing the Chart Before Installing](https://github.com/kubernetes/helm/blob/master/docs/using_helm.md#customizing-the-chart-before-installing).
+
+| Parameter                 | Description                                         | Default                           |
+|---------------------------|-----------------------------------------------------|-----------------------------------|
+| `image.repo`              | docker image                                        | intelsdi/snap4kube                |
+| `image.tag`               | docker image tag                                    | latest                            |
+| `image.pullPolicy`        | image pull policy                                   | IfNotPresent                      |
+| `mounts`                  | list of volumes to mount inside the Snap container  | /proc:/proc_host                  |
+| `host_autoload`           | host path to the directory with plugins and tasks   | /opt/autoload                     |
+| `privileged`              | security context needed to access files on the host | false                             |
+| `config.log_level`        | log level of Snap, (1=debug, 5=error)               | 3                                 |
+| `config.log_path`         | container path to Snap logs                         | /snap                             |
+| `config.autodiscover_path`| list of volumes to mount inside the Snap container  | /opt/snap/autoload                |
+| `autoload`                | list of Snap plugins/tasks URLs to download and load| empty                             |
+
+## 7. Loading Snap plugins
+
+The easiest way to download and load Snap plugins is to specify plugins binaries URL in the `autoload` parameter in `values.yaml` file.
+
+Here is a simple example of using [CPU collector plugin](https://github.com/intelsdi-x/snap-plugin-collector-cpu) with [file publisher plugin](https://github.com/intelsdi-x/snap-plugin-publisher-file).
+
+```
+$ cat <<EOF >> new_values.yaml
+autoload:
```

Comment:
That depends. In the present version of code:
- if the user specifies URLs, all the pods will get plugins/tasks specified in the values.yaml
- however there is a possibility to mount individual plugins/tasks per pod (node actually) with the use of hostPath.
Unfortunately, as @jose5918 and @thomastaylor312 have pointed out, using hostPath could be error prone in this case and we have to discuss other options.